docs/DesignOverview.md

```diff
@@ -119,14 +119,13 @@ Here are a couple of simple examples showing `Tensor` and `ShapedArray`:
 
 ```swift
 // `Tensor` examples.
-var matrix = Tensor<Float>(shape: [2, 2], scalars: [1, 2, 3, 4])
 var matrix: Tensor<Float> = [[1, 2], [3, 4]]
 // `matrix` represents [[1, 2], [3, 4]].
 
 // Arithmetic operations, using TensorFlow.
 let sum = matrix + matrix
-let sqrt = sqrt(matrix)
-let matrixProduct = matrix.dot(matrix)
+let sqrt = Raw.sqrt(matrix)
```

Comment:
This one didn't work because `sqrt` is colliding with the variable name. You can rename `let sqrt` to `let root` or something. Using `Raw` here is not recommended because there is a native Swift API for it, which is `sqrt(_:)`.

Reply:
kk, tweaked!